features/checkout/paying_for_order/paying_with_paypal_during_checkout.feature

```diff
@@ -16,27 +16,30 @@ Feature: Paying with paypal during checkout
         Given I am logged in as "john@example.com"
         And I added product "PHP T-Shirt" to the cart
         And I proceed selecting "PayPal Express Checkout" payment method
-        And I confirm my order with paypal payment
-        When I sign in to PayPal and pay successfully
+        When I confirm my order
+        And I try to pay
+        And I sign in to PayPal and pay successfully
         Then I should be redirected back to the thank you page
 
     @ui
     Scenario: Cancelling the payment
         Given I am logged in as "john@example.com"
         And I added product "PHP T-Shirt" to the cart
-        When I proceed selecting "PayPal Express Checkout" payment method
-        And I confirm my order with paypal payment
+        And I proceed selecting "PayPal Express Checkout" payment method
+        When I confirm my order
+        And I try to pay
         And I cancel my PayPal payment
-        Then I should be redirected back to the order payment page
+        Then I should be able to pay again
 
     @ui
     Scenario: Retrying the payment with success
         Given I am logged in as "john@example.com"
         And I added product "PHP T-Shirt" to the cart
         And I proceed selecting "PayPal Express Checkout" payment method
-        And I confirm my order with paypal payment
+        And I confirm my order
+        And I try to pay
         But I cancel my PayPal payment
-        And I should be redirected back to the order payment page
+        And I should be able to pay again
```

Comment:
`I should be able to pay again` should not be a part of `Given` block. It is assertion....